screens/AppNavigator/screens/Dex/DexRemoveLiquidity.tsx

```diff
@@ -0,0 +1,191 @@
+        </View>
+      </ScrollView>
+      <View style={tailwind('w-full h-16')}>
+        <ContinueButton
+          enabled={Number(percentage) !== 0}
+          onPress={removeLiquidity}
+        />
+      </View>
+    </View>
+  )
+}
+
+function AmountSlider (props: { current: number, onChange: (percentage: number) => void, viewStyle: StyleProp<ViewStyle> }): JSX.Element {
+  return (
+    <View style={[tailwind('flex-row items-center border-t border-gray-200'), props.viewStyle]}>
+      <TouchableOpacity
+        style={tailwind('mr-4')}
+        onPress={() => props.onChange(0)}
+      >
+        <Text style={tailwind('text-gray-500 text-xs')}>None</Text>
+      </TouchableOpacity>
+      <Slider
+        value={props.current}
+        minimumValue={0}
+        maximumValue={100}
+        minimumTrackTintColor='#ff00af'
+        thumbTintColor='#ff00af'
+        onValueChange={(val) => props.onChange(val)}
+      />
+      <TouchableOpacity
+        style={tailwind('ml-4')}
+        onPress={() => props.onChange(100)}
+      >
+        <Text style={tailwind('text-gray-500 text-xs')}>All</Text>
+      </TouchableOpacity>
+    </View>
+  )
+}
+
+function CoinAmountRow (props: { symbol: string, amount: BigNumber }): JSX.Element {
+  const TokenIcon = getTokenIcon(props.symbol)
+  return (
+    <View style={tailwind('flex-row items-center border-t border-gray-200 p-2')}>
+      <View style={tailwind('flex-row flex-1 items-center justify-start')}>
+        <TokenIcon style={tailwind('ml-2')} />
+        <Text style={tailwind('ml-2')}>{props.symbol}</Text>
+      </View>
+      <Text style={tailwind('flex-1 text-right text-gray-500 mr-2')}>{props.amount.toString()}</Text>
+    </View>
+  )
+}
+
+function PriceRow (props: { lhs: string, rhs: string[], rowStyle?: StyleProp<ViewStyle> }): JSX.Element {
+  return (
+    <View style={[tailwind('bg-white p-2 border-t border-gray-200 flex-row items-start w-full'), props.rowStyle]}>
+      <View style={tailwind('flex-1 ml-2')}>
+        <Text style={tailwind('font-medium')}>{props.lhs}</Text>
+      </View>
+      <View style={tailwind('flex-1 mr-2')}>
+        {props.rhs.map((val, idx) => (<Text key={idx} style={tailwind('font-medium text-right text-gray-500')}>{val}</Text>))}
+      </View>
+    </View>
+  )
+}
+
+function ContinueButton (props: { enabled: boolean, onPress: () => void }): JSX.Element {
+  const buttonColor = props.enabled ? PrimaryColorStyle.bg : { backgroundColor: 'gray' }
+  return (
+    <TouchableOpacity
+      disabled={!props.enabled}
+      style={[tailwind('m-2 p-3 rounded flex-row justify-center'), buttonColor]}
+      onPress={props.onPress}
+    >
+      <Text style={[tailwind('text-white font-bold')]}>Continue</Text>
+    </TouchableOpacity>
+  )
+}
+
+async function constructSignedRemoveLiqAndSend (whaleAPI: WhaleApiClient, account: WhaleWalletAccount, tokenId: number, amount: BigNumber): Promise<string> {
+  const feeRate = new WhaleFeeRateProvider(whaleAPI)
```

Comment:
Fuxing refactored the transaction builder to require fewer params. You can take a look here - https://github.com/DeFiCh/wallet/blob/0dcdca523d14a1e234371dff0e8633f11d74ce06/screens/AppNavigator/screens/BalancesScreen/SendScreen/SendScreen.tsx#L52